{{cookiecutter.project_slug}}/setup.py

```diff
@@ -32,6 +32,9 @@
     'gunicorn',
     {%- endif %}
     {%- endif %}
+    {%- if cookiecutter.has_anyblok_marshmallow == 'yes' %}
```

Comment:
Could be nice to not be casse sensitive on yes/no answers, I was wondering about [Internationalization](https://docs.python.org/3.5/library/locale.html#locale.YESEXPR) or using [dist.utils](https://docs.python.org/3.6/distutils/apiref.html?highlight=distutils.util#distutils.util.strtobool) but I'm a bit worried about create weakness ...

Anyway looks good!

Reply:
or add_anyblok_marshmallow

Reply:
+1 for Internalization